### ISSUE TITLE
incorrect use of params in documentation

### ISSUE THREAD

**Issue by Krastanov** [GenieFramework/Genie.jl#650]
In the current docs
https://genieframework.com/docs/genie/v5.11/guides/Interactive-environment.html#Handling-query-params

it suggests to use `get(params, ...)`, when it seems it should be `get(params(), ...)`

![image](https://github.com/GenieFramework/Genie.jl/assets/705248/8467d070-d0a5-44d7-8a8a-aa03d7ff8e0c)

**Comment by essenciary** [comment]
@Krastanov that's right, good catch - would you be comfortable fixing it and submitting a PR?
